### ISSUE TITLE
update master nodes causes explosion! 

### ISSUE THREAD

**Issue by DStorck** [samsung-cnct/k2#582]
If master nodes are updated, once they all are killed off - the cluster becomes unreachable and returns a fatal error. Seems like there are two options: 

- keep the playbook running after the nodes die and run a check to see when the nodes are reachable again - perhaps check api server for readiness

- If the cluster is going to be out of service for awhile during node re-creation, there could be a graceful exit and a nice message that tells the user their cluster will be out of commission for awhile and not to worry when `kubectl` returns `Unable to connect to the server: EOF`

```TASK [/root/k2/ansible/roles/kraken.update/kraken.update.aws : Delete and Terminate Nodes] ***************************************************************************************************************************************************
Terminating instance: i-012a6fbb0ecadaad3
Deleting node: ip-10-0-103-64.ec2.internal
Available Nodes: 3/3
Node has been updated.
ok: [localhost] => (item={'key': u'ip-10-0-103-64.ec2.internal', 'value': u'i-012a6fbb0ecadaad3'})
Terminating instance: i-0af2fddd21bb8d63d
Deleting node: ip-10-0-228-112.ec2.internal
Available Nodes: 3/3
Node has been updated.
ok: [localhost] => (item={'key': u'ip-10-0-228-112.ec2.internal', 'value': u'i-0af2fddd21bb8d63d'})
Terminating instance: i-046f41b849cbe488a
Deleting node: ip-10-0-48-16.ec2.internal
Exception when calling CoreV1Api->list_node: (0)
Reason: SSLError
EOF occurred in violation of protocol (_ssl.c:661)


fatal: [localhost]: FAILED! => {"failed": true, "msg": "Unexpected templating type error occurred on ({{ item.key |delete_and_terminate_node_filter(item.value, expected_node_count, kubeconfig, aws_region, nodepool.name) }}): int() argument must be a string or a number, not 'NoneType'"}```

**Comment by DStorck** [comment]
this seems like it can easily be fixed by doing a bit more error handling in this line : ` current_count = int(self.current_node_count(kubeconfig, nodepool_label)` of `delete_and_terminate_node_filter`.

**Comment by guineveresaenger** [comment]
addressed by: #773
